.github/workflows/node-4+.yml

```diff
@@ -81,6 +81,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
           after_install: npm run copy-metafiles && ./tests/dep-time-travel.sh
           skip-ls-check: true
+      - run: npm run pretest
```

Comment:
why is this needed here? pretest is `linklocal` which is meant only for local development.

Reply:
How else can I link the yet unreleased version of `eslint-plugin-utils` for tests?

Reply:
BTW `appveyor.yml` includes this script as well:
https://github.com/import-js/eslint-plugin-import/blob/main/appveyor.yml#L88

Reply:
An example of a failure: https://github.com/import-js/eslint-plugin-import/runs/3464305918

Reply:
Fair enough; I guess we don't change the utils package often enough for this to have been a problem.